generators/dockertools/templates/python/Dockerfile

```diff
@@ -15,10 +15,19 @@ RUN apk --update add --virtual build-dependencies libffi-dev openssl-dev python-
 
 COPY . /app
 <% if (!enable) { %>
+<% if (language === "PYTHON") { %>
 ENV FLASK_APP=server
+<% } else { %>
+ENV DJANGO_APP=<%= name %>
+<% } %>
 <% } %>
 <% if (enable) { %>
 CMD ["echo", "No run command specified in Dockerfile"]
 <% } else { %>
+<% if (language === "PYTHON") { %>
+CMD ["gunicorn", "-b", "0.0.0.0:<%= port %>", "server:app"]
```

Comment:
Why do we have two commands here? `python manage.py start` executes the `gunicorn` script, so this isn't necessary